internal/runners/export/env.go

```diff
@@ -35,26 +32,15 @@ func NewEnv(prime primeable) *Env {
 func (e *Env) Run() error {
 	rt, _, err := runtime.NewFromProject(e.project, target.TriggerActivate, e.analytics, e.svcModel, e.out, e.auth)
 	if err != nil {
-		if errs.Matches(err, &model.ErrNoMatchingPlatform{}) {
-			branches, err := model.BranchNamesForProjectFiltered(e.project.Owner(), e.project.Name(), e.project.BranchName())
-			if err == nil && len(branches) > 1 {
-				return locale.NewInputError("err_alternate_branches", "", e.project.BranchName(), strings.Join(branches, "\n - "))
-			}
-		}
-		if !authentication.LegacyGet().Authenticated() {
-			return locale.WrapError(err, "err_export_env_auth", "Could not update runtime files. If this is a private project ensure that you are authenticated.")
-		}
-		return locale.WrapError(err, "err_export_new_runtime", "Could not get new runtime")
+		return locale.WrapError(err, "err_export_new_runtime", "Could not initialize new runtime")
```

Comment:
```suggestion
		return locale.WrapError(err, "err_export_new_runtime", "Could not initialize  runtime")
```

This wouldn't be a new runtime. I guess you're referring to `NewFromProject()` which refers to the struct it's initializing; not something users need to be concerned with.